app/ui/legacy/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -1707,6 +1731,20 @@ private void setContextCapabilities(Account account, Menu menu) {
                     menu.findItem(R.id.spam).setVisible(false);
                 }
             }
+
+            // if currentFolder is outbox: reduce available menu items to 'move to draft' and 'delete'
```

Comment:
Experience has shown that comments quickly become outdated (code is changed, but comments aren't updated accordingly). That's why we try to use comments sparingly. When necessary we use them to describe *why* something is done, not *how* (the code is the how).
In this case I don't think a comment is necessary.